portal.go

```diff
@@ -1176,6 +1178,15 @@ func (portal *Portal) CreateMatrixRoom(user *User, groupInfo *types.GroupInfo, i
 	return nil
 }
 
+func (portal *Portal) addToSpace(spaceID id.RoomID, portalID id.RoomID) {
```

Comment:
This should also set the space's parent. (`m.space.parent`, or whatever they are calling it)

Reply:
We can choose one to two ways - either `m.space.child` or `m.space.parent`
From [MSC 1772](https://github.com/matrix-org/matrix-doc/blob/old_master/proposals/1772-groups-as-rooms.md#membership-of-spaces)
> The parent/child relationship can be expressed in one of two ways

Reply:
It's not an either or thing, both is the correct option for canonical relationships.

Reply:
@tulir So we need to run both 
1. `update space-room` by adding a child
2. `create a new room` by stating it's parent

I just checked element-web and only saw one request go through - `m.space.parent`. (Other requests are related to sync). Why does element-web only run te `m.space.parent` request?

Reply:
Done on 0fb2abb